frontend/public/components/pod.tsx

```diff
@@ -582,17 +748,23 @@ export const PodsPage = connect<{}, PodPagePropsFromDispatch, PodPageProps>(
       const id = setInterval(updateMetrics, 30 * 1000);
       return () => clearInterval(id);
     }
-  }, [namespace]);
+  }, [namespace, setPodMetrics]);
```

Comment:
I'm not sure this new dependency is right. Will this cause an infinite loop?

Reply:
@spadgett i didn't intentionally change this code...maybe a merge issue...removing as its unintentional.